Services/DataX.Flow/Flow.ManagementService/Controllers/FlowManagementController.cs

```diff
@@ -422,6 +422,49 @@ public async Task<ApiResult> RestartJob([FromBody] string jobName)
             }
         }
 
+        [HttpPost]
+        [Route("job/restartallwithretries")]
+        [DataXWriter]
+        public async Task<ApiResult> RestatAllWithRetries([FromBody] RestatAllWithRetriesRequest request)
```

Comment:
Instead of taking parameter of type RestatAllWithRetriesRequest, lets use JObject. See "public async Task<ApiResult> SaveFlow([FromBody]JObject config)" for reference